config/workflows/templates/17/job.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="urn:proactive:jobdescriptor:3.8"
+        xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+        name="Start At" projectName="1. Basic Workflows"
+        priority="normal"
+        onTaskError="continueJobExecution">
+
+  <description>
+    <![CDATA[ START_AT generic information is defined for Job Computing PI according to MonteCarlo method. ]]>
+  </description>
+  <genericInformation>
+    <info name="START_AT" value="2017-04-01T12:00:00+01:00"/>
```

Comment:
If the aim is to show that the workflow is still pending before the START_AT date then we should increase the default value so it will not be outdated in 7 days.